clients/python/test/e2e/conftest.py

```diff
@@ -107,6 +112,11 @@ def api_client() -> Iterable[osparc.ApiClient]:
             yield api_client
 
 
+@pytest.fixture(scope="session")
+def files_api(api_client: osparc.ApiClient) -> Iterable[osparc.FilesApi]:
+    yield osparc.FilesApi(api_client=api_client)
```

Comment:
you can simply return here. 

Are you perhaps trying to do this? 
```python
@pytest.fixture(scope="session")
def files_api(api_client: osparc.ApiClient) -> Iterable[osparc.FilesApi]:
    with osparc.FilesApi(api_client=api_client) as api:
               yield api
               
```
If the object returned by `osparc.FilesApi` has a context manager (i.e. defines `__enter__`, `__exit__`) then you can use the fixture to start and stop it, this would be equivalent to
```python
@pytest.fixture(scope="session")
def files_api(api_client: osparc.ApiClient) -> Iterable[osparc.FilesApi]:
    api = osparc.FilesApi(api_client=api_client)
    api.__enter__()
    yield api
    # catch exc etc ...
    api.__exit__(exc)
               
```

Reply:
Yes, I need to have the context manager for the `api_client`. I did it here too by mistake. I have fixed the mistake